controller_manager/src/controller_manager.cpp

```diff
@@ -1938,7 +1941,14 @@ void ControllerManager::list_hardware_interfaces_srv_cb(
   {
     controller_manager_msgs::msg::HardwareInterface hwi;
     hwi.name = command_interface_name;
-    hwi.is_available = resource_manager_->command_interface_is_available(command_interface_name);
```

Comment:
Hm I am not sure why I implemented that a reference interface is not available if the controller is active, it doesn't make sense. Can we update this in the RM actually?

This here will work for sure, and it is a good workaround, just not sure if there is no simpler way by changing interface status in RM

Reply:
Can we just call this method when activating the controller?

https://github.com/ros-controls/ros2_control/blob/master/controller_manager/src/controller_manager.cpp#L1412

then we don't need additional logic in the `list_hardware_interaces_srv_cb`. The test should stay, of course.

Reply:
Hello @destogl,

I've updated it so all the exported reference interfaces are available upon activation and then are unavailable upon deactivation.

Thank you